### PR TITLE
updated class diagrams to show static

### PR DESCRIPTION
**Pull request by PrinceCatt**

### CODE REVIEW

File: docs/FileHandler.puml [hunk 1]
```diff
@@ -2,16 +2,16 @@
 'https://plantuml.com/class-diagram
 
 class FileHandler {
-  +initFile(Records): void
```
Comment (yentheng0110) [inline docs/FileHandler.puml:5]: Please add some class level attributes in the class diagram too:
Below is copied from the code:
private static final Logger logger = Logger.getLogger(FileHandler.class.getName());
private static String filePath = "data" + File.separator + "bookbob_data.txt";
private static String appointmentFilePath = "data" + File.separator + "bookbob_appointment.txt";

File: docs/FileHandler.puml [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
+  +{static} initFile(Records): void
+  +{static} initFile(AppointmentRecord): void
+  +{static} convertPatientToOutputText(Records): String
+  +{static} convertPatientToOutputText(AppointmentRecord): String
+  +{static} autosave(Records): void
+  +{static} autosave(AppointmentRecord): void
+  +{static} retrieveData(Records): void
+  +{static} retrieveData(AppointmentRecord): void
+  +{static} parseVisitInputString(visitString): Visit
+  +{static} parseList(input): ArrayList<String>
```
Comment (yentheng0110) [inline docs/FileHandler.puml:14]: Good that you make the methods as class-level methods

**Review by yentheng0110 — CHANGES_REQUESTED** [review]
Overall LGTM, just need to add some class level attributes in the class diagram of FileHandler class

File: docs/FileHandler.puml (hunk 1 above)
Comment (PrinceCatt) [reply docs/FileHandler.puml:5]: noted

**Review by yentheng0110 — APPROVED** [review]

**Pull request merged**